### DISCUSSION TITLE
커밋 메시지 지라 티켓 번호 관련

### DISCUSSION THREAD

**Discussion by waterfogSW** [prgrms-web-devcourse/Team-Saiko-BidMarket-BE#4] [Announcements]
레포지토리 폴더에서 `npm install husky jira-prepare-commit-msg --save-dev && npx husky install` 명령어만 입력해 주시면 다음과 같이 브랜치의 티켓 번호에 따라 커밋 메시지가 자동화 됩니다.

![image](https://user-images.githubusercontent.com/28651727/180646768-1fd60901-1455-4e58-8e6f-d44f246bb4c8.png)

https://github.com/prgrms-web-devcourse/Team-Saiko-BidMarket-BE/pull/2

**Comment by dojinyou** [comment]
local에 작업 완료했습니다! 👍
